src/controller/python/test/test_scripts/cluster_objects.py

```diff
@@ -331,6 +331,16 @@ async def TriggerAndWaitForEvents(cls, devCtrl, req):
         await cls._TriggerEvent(devCtrl)
         await cls._RetryForContent(request=lambda: devCtrl.ReadEvent(nodeid=NODE_ID, events=req), until=lambda res: res != 0)
 
+    @classmethod
+    @base.test_case
+    async def TestGenerateUndefinedFabricScopedEventRequests(cls, devCtrl):
+        logger.info("Running TestGenerateUndefinedFabricScopedEventRequests")
+        await devCtrl.SendCommand(nodeid=NODE_ID, endpoint=1, payload=Clusters.TestCluster.Commands.TestEmitTestFabricScopedEventRequest(arg1=0))
+        res = await devCtrl.ReadEvent(nodeid=NODE_ID, events=[
+            (1, Clusters.TestCluster.Events.TestEvent, 0),
+        ])
```

Comment:
What is this actually testing? If it isn't testing anything meaningful, we shouldn't have it here.